continuous-integration/reference-architectures/README.md

```diff
@@ -1,3 +1,181 @@
 # Reference Architectures
 
-Nothing here yet, come back soon... 
\ No newline at end of file
+## Overview
+Architectures presented here provides a high-level reference guide for the implementation of 
+continuous integration best practices. They describe the structure, component interrelationship and 
+processes invoked without detailing any specific implementation. Please see categories and links 
+below for details.
+
+## Continuous Integration
```

Comment:
I'd also link to your CI starter kit somewhere in your ref architecture, to suggest to the reader they can quickly instantiate the ref architecture when needed.

Reply:
Also - add links to your starter kit and ref architecture to the Use Cases diagram

Reply:
Links added, but this will evolve over time as we approach new tasks in SLIM.